native/src/widget/pane_grid.rs

```diff
@@ -208,14 +208,15 @@ impl<'a, Message, Renderer> PaneGrid<'a, Message, Renderer> {
     /// Enables the resize interactions of the [`PaneGrid`], which will
     /// use the provided function to produce messages.
     ///
-    /// Panes can be resized using `Modifier keys + Right click`.
+    /// The `leeway` describes the amount of space around a split that can be
```

Comment:
Maybe clarification on how `leeway` is applied. Is it

- Space *on either side* of the split?
- Total space straddling the split, so you have `leeway / 2` on either side?

Reply:
I tried to clarify it in https://github.com/hecrj/iced/pull/397/commits/8b93c9cb6a623d8f936e7e7135a2598e3ce4e4ef.

Reply:
That's better :-)